pkg/controller/datavolume-controller.go

```diff
@@ -2480,3 +2510,12 @@ func GetRequiredSpace(filesystemOverhead float64, requestedSpace int64) int64 {
 func newLongTermCloneTokenGenerator(key *rsa.PrivateKey) token.Generator {
 	return token.NewGenerator(common.ExtendedCloneTokenIssuer, key, 10*365*24*time.Hour)
 }
+
+// GetUsableSpace calculates space to use taking file system overhead into account
+func GetUsableSpace(filesystemOverhead float64, availableSpace int64) int64 {
```

Comment:
Would it make sense to push this up into pkg/util/util.go so it can be used here and in data-processor.go which has the same function already.

Reply:
yes, and it would be good to move it together with the reverse function and I think it would be even better to create a filesystem.go file with constants and functions useful for filesystem storage. 

So I wanted to do it quick for the bug to have the bugfix faster. But I can try moving this function now If you think it can be qucik and painless.

Reply:
I think I can easily move it right now

Reply:
done